x/session/keeper/session_hydrator.go

```diff
@@ -237,3 +237,15 @@ func sha3Hash(bz []byte) []byte {
 	hasher.Write(bz)
 	return hasher.Sum(nil)
 }
+
+func SessionIdBzToString(appPubKey, serviceId, blockHash string, blockHeight int64) string {
```

Comment:
1. Godoc
2. Make sure to re-use in the actual on-chain code as well so there's only 1 source of truth for the implementaiton

Reply:
Please review its on-chain usage